File: Napakalaki.ruby/DeathBadConsequence.rb
#encoding: utf-8
require_relative 'NumericBadConsequence'
require_relative 'Player'

module NapakalakiGame

class DeathBadConsequence < NumericBadConsequence

  def initialize (texto)
    super(texto, Player.MAXLEVELS, BadConsequence.MAXTREASURES, BadConsequence.MAXTREASURES)
  end

end

end

File: Napakalaki.java/src/NapakalakiGame/DeathBadConsequence.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author victor
 */
public class DeathBadConsequence extends NumericBadConsequence {
    
    public DeathBadConsequence(String texto){
        super (texto, Player.MAXLEVEL, MAXTREASURES, MAXTREASURES);
    }
}
